pkg/keyidentifier/keyidentifier.go

```diff
@@ -15,13 +15,14 @@ import (
 )
 
 type KeyInfo struct {
-	Type       string // Key type. rsa/dsa/etc
-	Format     string // file format
-	Bits       int    // number of bits in the key
-	Encryption string // key encryption algorythem
-	Encrypted  *bool  // is the key encrypted
-	Comment    string // comments attached to the key
-	Parser     string // what parser we used to determine information
+	Type        string // Key type. rsa/dsa/etc
+	Format      string // file format
+	Bits        int    // number of bits in the key
+	Encryption  string // key encryption algorythem
+	Encrypted   *bool  // is the key encrypted
+	Comment     string // comments attached to the key
+	Parser      string // what parser we used to determine information
+	Fingerprint string // the fingerprint of the key, in SHA256 format
```

Comment:
hmm, I'm not sure on that either. Largely because I think that the fingerprint we report should match whatever `$TOOL` reports, where `$TOOL` is the standard program that would generally be used to get the fingerprint. It doesn't add a lot of value to report the md5 sha fingerprint of a modern openssh key, when the standard is the sha256 and the md5 is just for backwards compatibility. 

One option that could strike a nice balance between simplicity and extensibility is to _not_ export the Fingerprint attr, define a `Fingerprint()` method on `KeyInfo`, and then in the future we can add additional methods as we see fit. Right now we don't need additional fingerprint types (as we only report fingerprints modern openssh-gen'd keys), but we would still be able to add additional fingerprint types in the future. We can decide how to store the different fingerprints (array of objects, map etc) when we get to that point.

Does that seem like a reasonable strategy?

Reply:
The issue, is that different tools use different fingerprints. 

In the webui, github displays the md5 fingerpringts.

At the command line, ssh-keygen defaults to the sha256.

So I'm pretty sure we need both.

Reply:
Using a `.Fingerprint()` method is interesting. Doing that implies that we'd also keep private data to facilitate it.

Reply:
> In the webui, github displays the md5 fingerpringts.
>
> At the command line, ssh-keygen defaults to the sha256.

Ahh ok, that clears things up a bit then. Yea if that's the case we _should_ record both. I'd even say that we should expose both in the osquery table

Reply:
I'm thinking that we just have SHA256Fingerprint _and_ MD5Fingerprint on the KeyInfo type as well as columns for both in the table

Reply:
So you think we should have them as straight fields on the `KeyInfo` struct?  (the first suggestion above) I'm good with that. Though I prefer to name them `Fingerprint<Type>` so they sort together

Reply:
Yea that's a good idea!